CSS/Styles.css

```diff
@@ -380,9 +476,186 @@ textarea {
 .socialinks li {
   display: inline-block;
   padding: 0;
+  width: fit-content;
+  height: auto;
 }
 
 .socialinks img {
   height: 50%;
   width: auto;
 }
+
+@media screen and (min-width: 768px) {
+  .nav {
+    box-shadow: 0 0.5px 0 #dddfe6;
+    background-image: none;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navlinks {
+    display: flex;
+  }
+
+  .homesection {
+    margin-bottom: 60px;
+    flex-wrap: nowrap;
+    height: 100vh;
+  }
+
+  .homewrp,
+  .aboutwrpr {
+    width: 50%;
+    padding-left: 70px;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+
+  .maintitle {
+    width: 100%;
+  }
+
+  .mtitle {
+    font-size: 90px;
+    line-height: 90px;
+    padding: 0;
+  }
+
+  .maintxt {
+    width: 90%;
+  }
+
+  .mtext {
+    margin: 0;
+  }
+
+  .decoimg {
+    display: none;
+  }
+
+  .decoimgdsktp {
+    display: block;
+  }
+
+  .works {
+    padding-top: 120px;
+  }
```

Comment:
Your `works` section doesn't quite look like the one from the template design.

Here's your design:

<img width="721" alt="Screen Shot 2022-06-03 at 13 11 49" src="https://user-images.githubusercontent.com/82426682/171922194-bdbfa909-cbfe-4612-9db3-b6733971ddfd.png">

Here's the template design:

<img width="542" alt="Screen Shot 2022-06-03 at 13 12 17" src="https://user-images.githubusercontent.com/82426682/171922259-f9089c3e-426e-4b09-a648-4b0110152bad.png">

Kindly fix it.